randovania/games/prime1/logic_database/Impact Crater.txt

```diff
@@ -208,8 +208,10 @@ Extra - aabb: [-49.130936, -391.06018, 2.7536602, 70.97014, -331.07498, 78.62515
           Charge Beam or Combat (Intermediate)
           Any of the following:
               Shoot Power Beam
-              # Fight Without Power Beam
-              Charge Beam and Flamethrower and Missile ≥ 25 and Plasma Beam and Combat (Intermediate) and Knowledge (Intermediate)
+              All of the following:
+                  # Fight Without Power Beam
+                  Charge Beam and Flamethrower and Plasma Beam and Combat (Intermediate) and Knowledge (Intermediate)
+                  Missile ≥ 25 or Missile ≥ 50 or Disabled Hard Mode
```

Comment:
I believe this should read `missile 25 and disabled hardmode` or `missiles 50`. Applies to the 2 below as well.